wiki/Tournaments/CWC/1/en.md

```diff
@@ -157,91 +157,322 @@ This competition has come to an end and resulted in the following podium:
   - [TAKA respect for J.S.B - Ubertreffen (kiddly) \[Another\]](https://osu.ppy.sh/beatmapsets/15740#fruits/56830)
   - [LEAF XCEED Music Division - Fur Elise (Louis Cyphre) \[Champion\]](https://osu.ppy.sh/beatmapsets/23916#fruits/81594)
   - [Hatsune Miku - Unhappy Refrain (NatsumeRin) \[Rin\]](https://osu.ppy.sh/beatmapsets/30128#fruits/105534)
   - [Furries in a Blender - Ridorii (-Lennox-) \[Insane\]](https://osu.ppy.sh/beatmapsets/29727#fruits/98496)
   - [supercell - Rock 'n' Roll Nan Desu no (Garven) \[Insane\]](https://osu.ppy.sh/beatmapsets/27807#fruits/93021)
-  - [Rhapsody - Emerald Sword (Duoprism) \[Extreme\]](https://osu.ppy.sh/beatmapsets/3198#fruits/25580)
+  - [Rhapsody - Emerald Sword (Reikin) \[Extreme\]](https://osu.ppy.sh/beatmapsets/3198#fruits/25580)
   - [IOSYS - Border of Death (Nakagawa-Kanon) \[Lunatic\]](https://osu.ppy.sh/beatmapsets/14107#fruits/53380)
 - Tiebreaker
   - **[wa. remixed celas - Suishou-Sekai \~Fracture\~ (soulfear) \[Another\]](https://osu.ppy.sh/beatmapsets/28984#fruits/96358)**
 
-### Round 1
+### Group Stage week 1
 
 - NoMod
   - [DJ YOSHITAKA - FLOWER (TKiller) \[Intense\]](https://osu.ppy.sh/beatmapsets/29996#fruits/104635)
   - [IOSYS - The Lovely, Freezing, Tomboyish Bath, Cirno's Hot Spring (Mafiamaster) \[Hot Spring\]](https://osu.ppy.sh/beatmapsets/27107#fruits/95431)
   - [Syrsa - Mad Machine (Louis Cyphre) \[Champion\]](https://osu.ppy.sh/beatmapsets/33052#fruits/107875)
   - [yak\_won - Lucid (soulfear) \[Extra\]](https://osu.ppy.sh/beatmapsets/15898#fruits/85852)
   - [TuTuWan - Wo Ge Zai Guang Dian (v2b) \[Qi Shi Ma\]](https://osu.ppy.sh/beatmapsets/12034#fruits/52940)
-  - [SHK - Weep Irish (fartownik) \[Another\]](https://osu.ppy.sh/beatmapsets/28545#fruits/95360)
-  - [NH22 - Corrosion (Cecilia) \[Lunatic\]](https://osu.ppy.sh/beatmapsets/17044#fruits/60941)
+  - [SHK - Weep Irish (AngelHoney) \[Another\]](https://osu.ppy.sh/beatmapsets/28545#fruits/95360)
+  - [NH22 - Corrosion (Lena) \[Lunatic\]](https://osu.ppy.sh/beatmapsets/17044#fruits/60941)
   - [HTT - Girls in Wonderland (wcx19911123) \[Insane\]](https://osu.ppy.sh/beatmapsets/34427#fruits/111902)
   - [Len - U.N. Owen was her? (Louis Cyphre) \[Insane\]](https://osu.ppy.sh/beatmapsets/22699#fruits/78171)
 - Tiebreaker
-  - **[chum - Genesis of Aquarion (Starrodkirby86) \[KIRBY Mix\]](https://osu.ppy.sh/beatmapsets/16782#fruits/60858)**
+  - **[chum - Genesis of Aquarion (Lena) \[KIRBY Mix\]](https://osu.ppy.sh/beatmapsets/16782#fruits/60858)**
 
-## Ruleset
+## Match Results
+
+## Finals
+
+Saturday, 4 February 2012 (3rd Place Playoff):
+
+| Team 1 |  |  | Team 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| Team Nub | 4 | **5** | **Chile** | [#1](https://osu.ppy.sh/community/matches/2563701) |
+
+Sunday, 5 February 2012 (Grand Final):
+
+| Team 1 |  |  | Team 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| half manual player team | 2 | **5** | **China** | [#1](https://osu.ppy.sh/community/matches/2568176) |
```

Comment:
should these be in sentence case? this goes for "win by default" and other ones like it as well

Reply:
this is literally the team's name orz, it shouldn't be sentence cased at all :(

![image](https://user-images.githubusercontent.com/36564236/106345585-e5f3f200-62e3-11eb-89f2-10106e1d3dbb.png)